rules.bzl

```diff
@@ -15,11 +11,10 @@ def _buildbuddy_toolchain_impl(rctx):
             default_platform = "platform_darwin"
     elif rctx.os.name == "windows":
         default_platform = "platform_windows"
+    elif rctx.os.arch == "aarch64":
+        default_platform = "platform_linux_arm64"
     else:
-        if rctx.os.arch == "aarch64":
-            default_platform = "platform_linux_arm64"
-        else:
-            default_platform = "platform_linux_x86_64"
+        default_platform = "platform_linux_x86_64"
```

Comment:
wild, didn't know `buildifier` could do this kind of source transformation :exploding_head: